apps/ios/Shared/ContentView.swift

```diff
@@ -28,6 +28,17 @@ struct ContentView: View {
     @State private var showWhatsNew = false
     @State private var showChooseLAMode = false
     @State private var showSetPasscode = false
+    @State private var mainViewActionSheet: MainViewActionSheet? = nil
```

Comment:
Why does action sheet needs to be passed via state, when it can be just boolean?

Reply:
And even if it needs to become enum, why pass the actual view definition via the state, and not just plain enum?